### DISCUSSION TITLE
How to set global default configuration?

### DISCUSSION THREAD

**Discussion by ArtyCodingart** [react-hook-form/react-hook-form#10057] [Q&A] [answered]
For example, I want all useForm to be with configuration.

`
{
  mode: 'all',
  resetOptions: {
    keepDirtyValues: true,
    keepErrors: true,
  },
  shouldFocusError: true,
}
`

**Comment by Moshyfawn** [comment, marked as answer]
Export a default config and import it for every form is the only way.

It's common for forms to have isolated logic and creating a shared config as a feature is not feasible when it's easy to do in user-land.
